packages/core/contracts/insured-bridge/BridgePool.sol

```diff
@@ -409,39 +450,19 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20, MultiCaller
      * @param request disputed relay request params.
      */
     function priceDisputed(
+        // Unused, but the identifier should be same as bridgeAdmin.identifier()
```

Comment:
is there any case where this could be called in the callback on a different identifier?

Reply:
ye, that makes sence to me. I agree with your logic. I'm still not convinced that everything works exactly as I understand it does but for now I'm happy.